pydoctor/sphinx_ext/api_output.py

```diff
@@ -0,0 +1,69 @@
+"""
+Generate the API docs using pydoctor to be integrated into Sphinx build system.
+
+Inside the Sphinx conf.py file you need to define the following configuration options:
+
+* pydoctor_cwd: Absolute path in which the pydoctor build is trigerred.
+* pydoctor_args: A list with all the pydoctor command line arguments used to trigger the build.
+
+You must call pydoctor with `--quiet` argument
+as otherwise any extra output is converted into Sphinx warnings.
+"""
+import os
+from sphinx.util import logging
+from sphinx.errors import ConfigError
+
+from contextlib import redirect_stdout
+from io import StringIO
+from pydoctor import __version__
+from pydoctor.driver import main
```

Comment:
Please group imports in PEP8 style: standard lib, other libs, internal. I want to adopt `isort` at some point to get it exactly right everywhere, but in the mean time having at least the grouping helps making the code easier to read.

Reply:
I think that we should adopt isort and black  ASAP so that the developers could forget about this and just focus on the code.